packages/vuelidate/src/index.js

```diff
@@ -108,7 +110,13 @@ export function useVuelidate (validations, state, globalConfig = {}) {
   const validationResults = ref({})
   const resultsCache = new ResultsStorage()
 
-  const { childResults, sendValidationResultsToParent, removeValidationResultsFromParent } = nestedValidations({ $scope, $stopPropagation })
+  const {
+    childResults,
+    sendValidationResultsToParent,
+    removeValidationResultsFromParent
+  } = instance
+    ? nestedValidations({ $scope })
+    : { childResults: ref('') }
```

Comment:
Should this be a `ref('')`?

Reply:
It's a ref with an array by default. We just need something to fallback to. I guess we can do `ref([])`?

Reply:
Ah, then yeah I think keeping the types unified might be a good move.